### DISCUSSION TITLE
"azureAutoComplete" option does not set auto-complete on PR

### DISCUSSION THREAD

**Discussion by wizgob** [renovatebot/renovate#12925] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

29.30.0

### Please select which platform you are using if self-hosting.

Azure DevOps (dev.azure.com)

### Describe the bug

I'm using a self hosted renovate with azure dev ops and the option "azureAutoComplete": true

Renovate creates PR as expected but auto-complete is not set.

In addition, the azureAutoComplete is not visible in the documentation ( https://docs.renovatebot.com/configuration-options/ )

### Relevant debug logs

<details><summary>Logs</summary>

{"name":"renovate","hostname":"LT-0002","pid":13480,"level":20,"logContext":"QXwTau1WUwrjOHEiYOncF","repository":"TestRenovate","branch":"renovate/jasminewd2-2.x","msg":"There are 0 errors and 0 warnings","time":"2021-12-02T16:43:52.529Z","v":0}
{"name":"renovate","hostname":"LT-0002","pid":13480,"level":20,"logContext":"QXwTau1WUwrjOHEiYOncF","repository":"TestRenovate","branch":"renovate/jasminewd2-2.x","msg":"getBranchPr(renovate/jasminewd2-2.x)","time":"2021-12-02T16:43:52.529Z","v":0}
{"name":"renovate","hostname":"LT-0002","pid":13480,"level":20,"logContext":"QXwTau1WUwrjOHEiYOncF","repository":"TestRenovate","branch":"renovate/jasminewd2-2.x","msg":"getPrList()","time":"2021-12-02T16:43:52.529Z","v":0}
{"name":"renovate","hostname":"LT-0002","pid":13480,"level":20,"logContext":"QXwTau1WUwrjOHEiYOncF","repository":"TestRenovate","branch":"renovate/jasminewd2-2.x","prTitle":"Update dependency @types/jasminewd2 to ^2.0.10","msg":"Creating PR","time":"2021-12-02T16:43:52.543Z","v":0}
{"name":"renovate","hostname":"LT-0002","pid":13480,"level":30,"logContext":"QXwTau1WUwrjOHEiYOncF","repository":"TestRenovate","branch":"renovate/jasminewd2-2.x","pr":6751,"prTitle":"Update dependency @types/jasminewd2 to ^2.0.10","msg":"PR created","time":"2021-12-02T16:43:53.269Z","v":0}
{"name":"renovate","hostname":"LT-0002","pid":13480,"level":20,"logContext":"QXwTau1WUwrjOHEiYOncF","repository":"TestRenovate","branch":"renovate/jasminewd2-2.x","msg":"addAssigneesReviewers(pr=6751)","time":"2021-12-02T16:43:53.269Z","v":0}
{"name":"renovate","hostname":"LT-0002","pid":13480,"level":20,"logContext":"QXwTau1WUwrjOHEiYOncF","repository":"TestRenovate","branch":"renovate/jasminewd2-2.x","msg":"Created Pull Request #6751","time":"2021-12-02T16:43:53.269Z","v":0}
{"name":"renovate","hostname":"LT-0002","pid":13480,"level":20,"logContext":"QXwTau1WUwrjOHEiYOncF","repository":"TestRenovate","branch":"renovate/jasminewd2-2.x","msg":"PR is not configured for automerge","time":"2021-12-02T16:43:53.269Z","v":0}
</details>

### Have you created a minimal reproduction repository?

No reproduction repository

**Comment by rarkins** [comment, marked as answer]
Both of these are explained by the 28.0.0 release notes. Please always read them before doing a major upgrade.

**Comment by corentinaltepe** [comment]
For those reading this ticket and wondering what @rarkins was talking about.

Here is the [release note 28.0.0](https://github.com/renovatebot/renovate/releases/tag/28.0.0).

> **platform**: Azure autocomplete will now only be enabled when `automerge` is also enabled (to align with how GitLab automerge already worked). If you were using `azureAutoComplete` before without explicitly enabling Renovate `automerge` field then you will now need to do so. Platform automerge is now the default behavior whenever automerge is enabled.

My understanding being you should set both `automerge` and `azureAutoComplete` to true for Azure DevOps PR to be set as auto-complete.
